tools/thresholds.cpp

```diff
@@ -57,13 +60,17 @@ void thresholds(const std::string &index_filename,
         float threshold = 0.0;
         if (results.size() == k) {
             threshold = results.back().first;
+            if (quantized) {
+                threshold = dequantize(threshold) * wdata.index_max_term_weight();
```

Comment:
Why do we want to unquantize those? Don't we need to use quantized thresholds with a quantized index?